FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/base/CreateMessageFragment.kt

```diff
@@ -832,84 +812,86 @@ class CreateMessageFragment : BaseSyncFragment(), View.OnFocusChangeListener,
 
   private fun updateRecipients() {
     recipientsTo?.chipAndTokenValues?.let {
-      contactsViewModel.fetchAndUpdateInfoAboutContacts(
-        ContactEntity.Type.TO,
+      recipientsViewModel.fetchAndUpdateInfoAboutRecipients(
+        RecipientEntity.Type.TO,
         it
       )
     }
 
     if (layoutCc?.visibility == View.VISIBLE) {
       recipientsCc?.chipAndTokenValues?.let {
-        contactsViewModel.fetchAndUpdateInfoAboutContacts(
-          ContactEntity.Type.CC,
+        recipientsViewModel.fetchAndUpdateInfoAboutRecipients(
+          RecipientEntity.Type.CC,
           it
         )
       }
     } else {
       recipientsCc?.setText(null as CharSequence?)
-      pgpContactsCc?.clear()
+      recipientWithPubKeysCc?.clear()
     }
 
     if (layoutBcc?.visibility == View.VISIBLE) {
       recipientsBcc?.chipAndTokenValues?.let {
-        contactsViewModel.fetchAndUpdateInfoAboutContacts(
-          ContactEntity.Type.BCC,
+        recipientsViewModel.fetchAndUpdateInfoAboutRecipients(
+          RecipientEntity.Type.BCC,
           it
         )
       }
     } else {
       recipientsBcc?.setText(null as CharSequence?)
-      pgpContactsBcc?.clear()
+      recipientWithPubKeysBcc?.clear()
     }
   }
 
   /**
-   * Run an action to update information about some [PgpContact]s.
+   * Run an action to update information about some [RecipientWithPubKeys]s.
    *
-   * @param pgpContacts Old [PgpContact]s
+   * @param recipients Old [RecipientWithPubKeys]s
    * @param progressBar A [ProgressBar] which is showing an action progress.
-   * @param type        A type of contacts
+   * @param type        A type of recipients
    * @param hasFocus    A value which indicates the view focus.
-   * @return A modified contacts list.
+   * @return A modified recipients list.
    */
-  private fun runUpdatePgpContactsAction(
-    pgpContacts: MutableList<PgpContact>?, progressBar: View?,
-    type: ContactEntity.Type, hasFocus: Boolean
-  ): List<PgpContact>? {
+  private fun runUpdateActionForRecipients(
+    recipients: MutableList<RecipientWithPubKeys>?,
+    progressBar: View?,
+    type: RecipientEntity.Type,
+    hasFocus: Boolean
+  ): List<RecipientWithPubKeys>? {
     if (listener.msgEncryptionType === MessageEncryptionType.ENCRYPTED) {
       if (hasFocus) {
-        pgpContacts?.clear()
+        recipients?.clear()
       } else {
-        if (isContactsUpdateEnabled) {
+        if (isRecipientsUpdateEnabled) {
```

Comment:
How is `isRecipientsUpdateEnabled` used here?

Reply:
It relates to old code. Removed.